src/atoms/Logo/index.jsx

```diff
@@ -14,7 +14,6 @@ type LogoType = React$Element<'span'> | React$Element<typeof Image>;
 
 function Logo({ src, alt, text, ...rest }: Props): LogoType {
   const props = omit(rest, 'src', 'alt', 'text');
```

Comment:
This is more of a question since I am not sure about this. But since the `...rest` gathers up all the remaining properties of the incoming props object which were not destructured into their own variables, isn't the `omit` function's usage incorrect here, by trying to omit some properties the `rest` object does not even contain?